mix.exs

```diff
@@ -33,16 +35,18 @@ defmodule Arc.Ecto.Mixfile do
   end
 
   defp package do
-    [maintainers: ["Sean Stavropoulos"],
-     licenses: ["Apache 2.0"],
-     links: %{"GitHub" => "https://github.com/stavro/arc_ecto"},
-     files: ~w(mix.exs README.md lib)]
+    [
+      maintainers: ["Sean Stavropoulos"],
+      licenses: ["Apache 2.0"],
+      links: %{"GitHub" => "https://github.com/stavro/arc_ecto"},
+      files: ~w(mix.exs README.md lib)
+    ]
   end
 
   defp deps do
     [
-      {:arc,  "~> 0.11.0"},
-      {:ecto, "~> 2.1"},
+      {:arc, "~> 0.11.0"},
+      {:ecto, "~> 3.0.0-rc.1"},
```

Comment:
Thanks @tarzan. Ecto 3.0 is now released. I've amended your commit in #101.

Looks good to me.